whiteBoarding/bubbleSort/whiteboardingcode

```diff
@@ -0,0 +1,29 @@
+// Whiteboarding Question, Monday, Oct. 30, 2018
+// Given an array, sort the array using bubbleSort and then return the array.
+
+function myBubble[arr] => {
```

Comment:
`function myBubble[arr] => {`. should be `function myBubble(arr) {` or `const myBubble = (arr) => { `.  Square brackets declare an array, parentheses declare a function `function someName() {}` or you can declare a function anonymously `function() {}` <- this only works in a callback format